### PR TITLE
StoplightProject Doesn't Have Layout (Yet)

### PR DESCRIPTION
**Pull request by philsturgeon**
- docs: fixed README link to React
- docs: removed layout from StoplightProject It does not support layout yet.

We might bring in a stacked layout for StoplightProject but we have to figure out how it looks first. https://github.com/stoplightio/elements/issues/1368

### CODE REVIEW

**Review by mpodlasin — APPROVED** [review]
Yup, 💯 correct

File: README.md [hunk 1]
```diff
@@ -58,12 +58,11 @@ import { API } from "@stoplight/elements";
 
 <API
   apiDescriptionUrl="https://api.apis.guru/v2/specs/github.com/1.1.4/openapi.yaml"
-  layout="traditional"
```
Comment (mmiask) [inline README.md:61]: Although there is no `traditional` layout in the `API` component, we still have `sidebar | stacked`

Another thing is that we don't have any description of `StoplightProject` in the main README yet. Do you think we should add it there?

**Review by mmiask — COMMENTED** [review]
Good eye with catching this `layout` inaccuracy  in `StoplightProject` docs. I just left one comment but overall this looks good 👍

File: README.md (hunk 1 above)
Comment (philsturgeon) [reply README.md:61]: Nah, we push people to docs and docs takes care of that.

**Pull request merged**